hydra/test_utils/launcher_common_tests.py

```diff
@@ -538,12 +607,23 @@ def test_to_absolute_path_multirun(
             overrides=overrides,
             prints="hydra.utils.to_absolute_path('/foo/bar')",
             expected_outputs=path,
+            post_run_callback=self.post_run_callback,
         )
-        outputs = [str(tmpdir / "foo/bar"), str(tmpdir / expected_dir)]
+        working_dir = (
+            self.get_test_app_working_dir()
+            if self.get_test_app_working_dir()
```

Comment:
is get_app_working_dir() specific to multirun?

Reply:
No,  I don't think so. But launcher always run in multi run mode, right?

Reply:
okay, is this actually hydra.run.dir in the config?

Reply:
no. ```hydra.runtime.cwd``` here is the tests' working dir (local machine dir, in this case the ```temp_dir```), but ```get_test_app_working_dir ``` is where the test app is actually kicked off, this is different on a remote machine. (in Ray's case, this value would be "/home/ubuntu" but in the local machine case this would be "/tmp/hydra_test/" or something like this.  )